templates/sqoop-parquet-full-load/avro-table-create.sql

```diff
@@ -0,0 +1,31 @@
+{#  Copyright 2017 Cargill Incorporated
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License. #}
+
+-- Create a Parquet table in Impala
+set sync_ddl=1;
+USE {{ conf.staging_database.name }};
+CREATE EXTERNAL TABLE IF NOT EXISTS {{ table.destination.name.replace('/','_').replace('.','_') }}_avro (
```

Comment:
You could put the name with the backslashes replaced into a variable then re-use it instead of doing the replace several times in a script. Another option would be to put a replaced version of the name into your tables.yml like 'table.destination.name.clean'. That way you have less repeated logic in the templates and it would be cleaner.